frontend/src/Buildings/buildingItem.jsx

```diff
@@ -1,56 +1,89 @@
 import React from 'react';
 import {
   View, TouchableHighlight,
-  Image, Text, Dimensions,
+  Image, Text,
   StyleSheet,
 } from 'react-native';
 import PropTypes from 'prop-types';
+import Colors from '../common/colors';
+import getIconImage from './assets';
 
-const buildingItemStyles = StyleSheet.create({
-  itemStyle: {
-    width: Dimensions.get('screen').width / 3,
+const styles = StyleSheet.create({
+  itemContainer: {
+    margin: 5,
+    padding: 24,
+    paddingBottom: 0,
+    backgroundColor: '#ffffff66',
+    borderRadius: 10,
     alignItems: 'center',
-    justifyContent: 'center',
-    paddingVertical: 10,
+    borderWidth: StyleSheet.hairlineWidth,
+    borderColor: 'rgba(0,0,0,.12)',
+    flexBasis: 120,
+  },
+  imageContainer: {
+    position: 'relative',
   },
   iconStyle: {
-    width: 'auto',
-    height: 'auto',
-    maxWidth: 200,
-    maxHeight: 200,
-    minWidth: 70,
-    minHeight: 70,
+    width: 64,
+    height: 64,
+  },
+  levelContainer: {
+    paddingVertical: 2,
+    paddingHorizontal: 6,
+    position: 'absolute',
+    top: -8,
+    right: -12,
+    borderRadius: 10,
+    backgroundColor: Colors.blueColor,
+  },
+  levelStyle: {
+    fontSize: 10,
+    fontWeight: 'bold',
+    color: Colors.whiteColor,
   },
-  textStyle: {
-    fontSize: 15,
-    lineHeight: 20,
+  titleStyle: {
+    fontSize: 14,
+    fontWeight: 'bold',
+    color: Colors.blackColor,
+  },
+  textContainer: {
+    paddingVertical: 7,
+    alignItems: 'center',
   },
 });
 
 function BuildingItem({
-  type, level, onPress, getIconImage,
+  type, level, onPress,
 }) {
   return (
-    <View style={buildingItemStyles.itemStyle}>
-      <TouchableHighlight underlayColor="transparent" onPress={onPress}>
-        <Image style={buildingItemStyles.iconStyle} source={getIconImage(type)} />
-      </TouchableHighlight>
-      <Text style={buildingItemStyles.textStyle}>{type}</Text>
-      <Text style={buildingItemStyles.textStyle}>{`Level ${level}`}</Text>
-    </View>
+    <TouchableHighlight
+      underlayColor="transparent"
+      onPress={onPress}
+      style={{ flex: 1, flexBasis: 120 }}
+    >
+      <View style={styles.itemContainer}>
+        <View style={styles.imageContainer}>
+          <Image style={styles.iconStyle} source={getIconImage(type)} />
+          <View style={styles.textContainer}>
+            <Text style={styles.titleStyle}>{type}</Text>
+          </View>
+          <View style={styles.levelContainer}>
+            <Text style={styles.levelStyle}>{`Lv.${level}`}</Text>
```

Comment:
```suggestion
            <Text style={styles.levelStyle}>{`Lvl ${level}`}</Text>
```

Lvl is the semi-official abbreviation for level:)